Use print() function in both Python 2 and Python 3

flake8 testing of https://github.com/easybuilders/easybuild on Python 3.8.0

__flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics__
```
./docs/scripts/gen_easyblocks_docs.py:69:11: E999 SyntaxError: invalid syntax
    print '%s updated' % so.options.out_file
          ^
./docs/scripts/gen_api_docs.py:52:7: E999 SyntaxError: invalid syntax
print "$ %s" % cmd
      ^
2     E999 SyntaxError: invalid syntax
2
```

--- a/docs/scripts/gen_api_docs.py
+++ b/docs/scripts/gen_api_docs.py
@@ -28,6 +28,8 @@ This script requires Sphinx to be installed (http://sphinx-doc.org/index.html).
 
 @author Caroline De Brouwer (Ghent University)
 """
+from __future__ import print_function
+
 import os
 import subprocess
 
@@ -49,5 +51,5 @@ if so.options.module is None:
 # -f forces to overwrite existing files
 # -e specifies that every module is written in a separate file
 cmd = "sphinx-apidoc -o %s %s -f -e" % (so.options.out_folder, so.options.module)
-print "$ %s" % cmd
+print("$ %s" % cmd)
 subprocess.call(cmd, shell=True)

--- a/docs/scripts/gen_easyblocks_docs.py
+++ b/docs/scripts/gen_easyblocks_docs.py
@@ -28,6 +28,8 @@ Generates documentation for (generic) easyblocks in .rst format
 @author Caroline De Brouwer (Ghent University)
 """
 
+from __future__ import print_function
+
 import os
 import easybuild.tools.config as config
 
@@ -66,7 +68,7 @@ easyblocks_overview = gen_easyblocks_overview_rst(so.options.module, so.options.
 txt = '\n'.join(autogen_comment + easyblocks_overview)
 if so.options.out_file:
     write_file(so.options.out_file, txt)
-    print '%s updated' % so.options.out_file
+    print('%s updated' % so.options.out_file)
 else:
-    print txt
+    print(txt)
 

